DistributionPackages/Neos.NeosConIo/Resources/Private/Scss/Organisms/_Talk.scss

```diff
@@ -30,7 +30,9 @@
   .speaker__speakerList {
     display: block;
     margin: 0;
-    width: $imageTeaser--speaker__width;
+    flex-basis: #{$base-line-height * 12}px;
```

Comment:
Why is it exactly 12 and why not use the variable from before?

Reply:
The variable resolves to 200px, which seemed to thin (see screenshot). So i used the same calculation as for the teaser image height, to make the image square (https://github.com/neos/Neos.NeosIo/blob/main/DistributionPackages/Neos.NeosIo/Resources/Private/Scss/Molecules/_ImageTeaser.scss#L6).

![grafik](https://user-images.githubusercontent.com/36864084/233296570-a8f4c28a-56a9-4f51-91ab-ffb2453763bd.png)

Reply:
Ah ok cool